scripts/_lines_of_code.py

```diff
@@ -45,50 +45,90 @@ def execute_bind(id_, message):
 class NotContributedError(Exception):
     pass
 
-
+excepted_files = [
+    'package.json',
+    'package-lock.json',
+    '.gitignore',
+    'yarn.lock'
+]
 class ContributedRepository:
     def __init__(self, contributed_to: Repository, user):
         self.total_lines_of_addition_in_contribution = 0
         self.total_lines_of_deletion_in_contribution = 0
+        self.excepted_lines_of_addition_in_contribution = 0
+        self.excepted_lines_of_deletion_in_contribution = 0
         self.name = contributed_to.full_name
+
         for contribution in contributed_to.get_stats_contributors():
             if contribution.author.id == user.id:
                 for week in contribution.weeks:
                     self.total_lines_of_addition_in_contribution += week.a
-                    self.total_lines_of_deletion_in_contribution += week.d
+                    self.total_lines_of_deletion_in_contribution += week.dRepository
```

Comment:
NameError is raised because, name 'week.dRepository' is not defined it should be 'week.d'